package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-formidable",
   "description": "A set of default eslint configurations from formidable",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "authors": "eric.baer@formidable.com, ryan.roemer@formidable.com, kyle.cesmat@formidable.com",
```

Comment:
Side not, I highly recommend using [np](https://www.npmjs.com/package/np) for publishing. I've been using it for WML component repos lately and it's much better than doing all the incantations yourself.